app/views/partials/foyer/personnes-a-charge.html

```diff
@@ -2,7 +2,7 @@
   <h1>Les enfants de votre foyer</h1>
 
   <div class="help-block">
-    <p>Les personnes de <span data-tooltip="Les personnes à votre charge de plus de 25 ans ne changent pas votre éligibilité aux aides calculées par ce simulateur.">moins de 25 ans</span> dont vous assumez la responsabilité, même sans lien de parenté.</p>
+    <p>Les personnes de <span data-tooltip="Les personnes à votre charge de plus de 25 ans non handicapées ne changent pas votre éligibilité aux aides calculées par ce simulateur.">moins de 25 ans </span> ou handicapées dont vous assumez la responsabilité, même sans lien de parenté.</p>
```

Comment:
Space introduced before `</span>`, will present an ugly `border-bottom` hanging out of nowhere.